### DISCUSSION TITLE
Regarding questions related to the RV64 assembly instruction portion of the code

### DISCUSSION THREAD

**Discussion by ZephyrLi-pro** [ptitSeb/box64#971] [Q&A]
I would like to inquire about how, in the rv64_next.S file, after the 'jr a3' instruction is executed, the program subsequently returns to the 'LinkNext' function.

**Comment by ptitSeb** [comment]
That's the magic of "jump table". I plan to write a blog post about that on box86.org. Hopefully it will done this month.

**Reply by ZephyrLi-pro** [reply]
Great! Looking forward to seeing it soon.
